src/shred_network/service.zig

```diff
@@ -203,6 +207,20 @@ pub fn start(
         .{repair_svc},
     );
 
+    if (conf.dump_shred_tracker) {
+        try service_manager.spawn("dump shred tracker", struct {
```

Comment:
this is a good tmp solution but i feel like maybe we can push these to geyser features later? (same as the gossip dump) maybe as a future ticket?

Reply:
I don't understand. How does this relate to geyser?

Reply:
im guessing the main reason you would want to dump shred information is to debug the information externally / while the process is still running - you can dump this information in a text file while it runs in a separate thread (ie, what we do in gossip) - or just stream the data out of the process using geyser and then having a reading process handle the rest (which is what im thinking - at least for a future ticket)

Reply:
oh that makes sense. I wasn't thinking of geyser so generally. That's a good idea but it would require a more complex setup in order to debug the code